src/io/readImageDICOMArrayBufferSeries.ts

```diff
@@ -78,11 +80,29 @@ const seriesBlockSize = 8
 
 const readImageDICOMArrayBufferSeries = async (
   arrayBuffers: ArrayBuffer[],
-  singleSortedSeries = false,
-  fileNames?: string[]
+  options?: ReadImageDICOMArrayBufferSeriesOptions | boolean,
+  fileNamesBackwardsCompatibility?: string[]
 ): Promise<ReadImageFileSeriesResult> => {
-  const validFileNames = (fileNames != null) && fileNames.length === arrayBuffers.length
+  let singleSortedSeries = false
+  let fileNames: undefined | string[]
+  if (typeof options === 'boolean') {
+    // Backwards compatibility
+    singleSortedSeries = options
+  }
+  if (typeof fileNamesBackwardsCompatibility !== 'undefined') {
+    fileNames = fileNamesBackwardsCompatibility
+  }
+  if (typeof options === 'object') {
+    if (typeof options.singleSortedSeries !== 'undefined') {
```

Comment:
Should `options.singleSortedSeries` type be enforced as boolean?

Reply:
Yes, this is already enforced via Typescript at compile time. This helps remove overhead of a run-time check.